server/router/auth.ts

```diff
@@ -18,22 +18,58 @@ const validIDs = [
   '322014396101-8u88pc3q00v6dre4doa64psr9349bhum.apps.googleusercontent.com',
 ];
 
-async function verify(clientID: string, token: string): Promise<void> {
+async function verify(clientID: string, token: string): Promise<LoginTicket> {
   const client = new OAuth2Client(clientID);
-  client.verifyIdToken({
+  const authRes = await client.verifyIdToken({
     idToken: token,
     audience: validIDs,
-  })
-    .then((ticket) => ticket.getPayload())
-    .catch((err) => { throw err; });
+  });
+  return authRes;
 }
 
 // Verify an authentication token
-router.post('/', async (req, res) => {
-  const { token, clientID } = req.body;
-  verify(token, clientID)
-    .then(() => res.send({ success: true }))
-    .catch(() => res.send({ success: false }));
+router.post('/', (req, res) => {
+  const {
+    token, clientID, userType, email,
+  } = req.body;
```

Comment:
Why not have this const declaration all on one line?

Reply:
eslint didn't like it, i can disable that style in the config